components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/impl/UserAwareAPIStoreTestCase.java

```diff
@@ -1,192 +0,0 @@
-/*
```

Comment:
UserAwareApiStore had some overriding methods with validations which seem to be unwanted. These methods have been removed in this PR. Since there are no methods in this class the test class was also removed.